src/Repeater/DefaultRepeaterLauncher.ts

```diff
@@ -126,7 +126,7 @@ export class DefaultRepeaterLauncher implements RepeaterLauncher {
       await this.startupManager.run(() => this.close());
     }
 
-    logger.log('Starting the Repeater (%s)...', this.info.version);
+    logger.log('Starting the Legacy Repeater (%s)...', this.info.version);
```

Comment:
Please avoid from change the log messages, keeping them as is. However, you may add a warning message to notify users regarding the utilization of the legacy flow (i.e. `--legacy`):


```ts
    logger.log('Starting the Repeater (%s)...', this.info.version);
    logger.warn('WARNING: You are using the legacy flow. Consider using the new repeater that utilizes standard ports.');

```